src/signals/incident/components/form/MapSelect/index.js

```diff
@@ -39,8 +38,7 @@ const MapSelectFormComponent = ({
   };
 
   const latlng = getLatlng(parent.meta);
-  const apiRoot = CONFIGURATION.API_ROOT_MAPSERVER;
-  const url = apiRoot + meta.endpoint;
+  const url = meta.endpoint;
```

Comment:
Same here. This makes a test fail.

Reply:
Ah, I didn't have a look at the tests yet.